addons/default-bitmap/userscript.js

```diff
@@ -0,0 +1,27 @@
+export default async function ({ addon, console }) {
+  const vm = addon.tab.traps.vm;
+  const _addCostume = vm.addCostume;
+  vm.addCostume = (md5, costume, ...args) => {
+    if (md5 === "cd21514d0531fdffb22204e0ec5ed84a.svg" && !addon.self.disabled) {
+      return _addCostume.call(
+        vm,
```

Comment:
We usually just write `this` here, but in this case it's equivalent.
For a second, I thought the VM was a parameter, though. It's a bit more confusing.

Also, if you want to follow conventions, the `_addCostume` variable should be named `originalAddCostume`.

Reply:
> Also, if you want to follow conventions, the `_addCostume` variable should be named `originalAddCostume`.

There's no convention - some addons use the `original` prefix, some use `old` and a few use an underscore.

Reply:
yeah, it's not strictly enforced